content/en/blog/2023/kwok/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Introducing KWOK for Istio Control Plane Testing"
+description: "Looking for a reliable way to test your Istio control plane at scale? Have you ever wondered how you could set up a cluster of thousands of nodes, simulate real nodes with a low resource footprint, and test at scale without spending a lot of money on infrastructure? Look no further! KWOK makes it easy."
+publishdate: 2023-07-20
+attribution: "Shiming Zhang (DaoCloud)"
+keywords: [Istio, kwok, testing]
+---
+
+Looking for a reliable way to test your Istio control plane at scale?
+Have you ever wondered how you could set up a cluster of thousands of nodes,
+simulate real nodes with a low resource footprint, and test at scale without spending a lot of money on infrastructure?
+Look no further! KWOK makes it easy.
+
+In this article, we'll introduce KWOK and show you how to test Istio with KWOK.
+
+## What is KWOK?
+
+KWOK stands for Kubernetes WithOut Kubelet. So far, it provides two tools:
+
+`kwok`
+: `kwok` is the cornerstone of this project, responsible for simulating the lifecycle of
+fake nodes, pods, and other Kubernetes API resources.
```

Comment:
Looking at the page, this section seems like it could be formatted differently. Just placing the tools above the description looks out of place. Maybe as a bullet point with a description like the advantages.